core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -154,26 +159,30 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     waitForTopics(client, List(), topics)
   }
 
-  @Test
-  def testMetadataRefresh(): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testMetadataRefresh(quorum: String): Unit = {
```

Comment:
As you said, we don't have controller for Kraft mode, I don't think it makes sense to enable it here since there's no test for Kraft. Could we create a JIRA ticket for this issue, that to add metadata refresh test for Kraft mode? This way, we won't forget it.

Reply:
I created KAFKA-13910 for this.